TessellationAndVoxelizationGeometryLibrary/Enclosure Operations/ConvexHull2D.cs

```diff
@@ -197,6 +214,7 @@ public static List<Point> ConvexHull2DMaximal(IList<Point> points)
                      * It is only possible for the IVertexConvHull to be outside one of the 3 to 8 edges, so once we
                      * add it, we break out of the inner loop (gotta save time where we can!). */
                     if (StarMath.crossProduct2(edgeUnitVectors[j], b) > 0) continue;
+                    if (StarMath.dotProduct(edgeUnitVectors[j], b) < 0) continue;
```

Comment:
That is the maximal convex hull. I didn't make any changes on it. Only the
minimal.

On Saturday, April 16, 2016, Steven Lindberg <notifications@github.com>
wrote:

> In TessellationAndVoxelizationGeometryLibrary/Enclosure
> Operations/ConvexHull2D.cs
> <https://github.com/DesignEngrLab/TVGL/pull/1#discussion_r59969872>:
>
> > @@ -197,6 +214,7 @@ public static List<Point> ConvexHull2DMaximal(IList<Point> points)
> >                       * It is only possible for the IVertexConvHull to be outside one of the 3 to 8 edges, so once we
> >                       * add it, we break out of the inner loop (gotta save time where we can!). */
> >                      if (StarMath.crossProduct2(edgeUnitVectors[j], b) > 0) continue;
> > +                    if (StarMath.dotProduct(edgeUnitVectors[j], b) < 0) continue;
>
> Whyyyy
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/DesignEngrLab/TVGL/pull/1/files/34fe9512e4bca7bb8a42fa19229be7983f879d17#r59969872>
>